lib/checklist.rb

```diff
@@ -6,4 +6,16 @@ def initialize(checklist_data)
   def name
     @checklist_data["name"]
   end
+
+  def checklist_items
+    @checklist_data["checkItems"]
+  end
+
+  def done_tasks
+    checklist_items.count { |list_item| list_item["state"] == "complete" }
+  end
+
+  def tasks
+    checklist_items.count
+  end
```

Comment:
These methods are being tested though card_spec but maybe it is a good idea to introduce the checklist_spec now that we have some methods to test.

Reply:
@jimmykarily can you maybe create an issue for this? Then we can finally merge this, and this is not strictly needed for this PR. :wink: